Blish HUD/GameServices/GameIntegration/Gw2InstanceIntegration.cs

```diff
@@ -5,6 +5,7 @@
 using System.IO;
 using System.Security.AccessControl;
 using System.Windows.Forms;
+using Blish_HUD._Utils;
```

Comment:
Once the util's namespace (mentioned below on that new class) is revised, we'll want to remember to delete this line.